### ISSUE TITLE
HttpApiIdForApigThrottling error

### ISSUE THREAD

**Issue by stanislav-web** [DianaIonita/serverless-api-gateway-throttling#25]
After update from 1.2.2 -> to 2.0.0 i am catching error `Could not find stack output variable named HttpApiIdForApigThrottling` while `sls deploy`

Throttling has configured on root level
```
custom:
  apiGatewayThrottling:
    maxRequestsPerSecond: 1000
    maxConcurrentRequests: 500
```

**Comment by thejuan** [comment]
Just came to report the same thing. `updateHttpApi` assumes there is an http api

```
[serverless-api-gateway-throttling] Could not find stack output variable named HttpApiIdForApigThrottling.
[serverless-api-gateway-throttling] No HTTP API (API Gateway v2) found. Throttling settings will be ignored for HTTP API endpoints.

... 

[serverless-api-gateway-throttling] Updating API Gateway REST API throttling settings (1 of 1).
[serverless-api-gateway-throttling] Done updating API Gateway REST API throttling settings.
[serverless-api-gateway-throttling] Could not find stack output variable named HttpApiIdForApigThrottling.
[serverless-api-gateway-throttling] Updating API Gateway HTTP API throttling settings...

...

Error:
Missing required key 'ApiId' in params

```

**Comment by DianaIonita** [comment]
Hi @stanislav-web and @thejuan,

Thanks for reporting the issue. I just released version [2.0.1](https://github.com/DianaIonita/serverless-api-gateway-throttling/releases/tag/2.0.1) which should fix it. Please reopen the ticket if you're still having problems.
